src/i_oalsound.c

```diff
@@ -469,6 +551,8 @@ void I_OAL_BindSoundVariables(void)
     BIND_NUM(snd_doppler, 0, 0, 10,
         "[OpenAL 3D] Doppler effect (0 = Off; 10 = Max)");
     BIND_BOOL(snd_limiter, false, "Use sound output limiter");
+    M_BindStr("snd_equalizer", &snd_equalizer, EQUALIZER_DEFAULT,
+        wad_no, "Equalizer gain percent (low mid1 mid2 high)");
```

Comment:
I'm curious how this interacts with the settings above it (headphones mode, air absorption, Doppler effect, resampler). I'm sure it's fine, there's a lot of powerful OpenAL features we don't use that "just work".